openwisp_utils/admin_theme/templates/openwisp_utils/email_template.html

```diff
@@ -172,9 +213,12 @@
       <div class="subtitle">{{ subtitle }}</div>
       {% endif %}
       {% endblock mail_header %}
-      {% block mail_body %}
-      <div class="msg">{{ message | safe }}</div>
-      {% endblock mail_body %}
+      <hr>
```

Comment:
I can't see this separator in my tests, this is a mobile screenshot:

![photo_5825572737421265469_y](https://github.com/user-attachments/assets/4f375fd9-3ca3-4f05-83af-0297383a2f70)

It looks good also without it, I think we could get rid of it to simplify.